test/bot.coffee

```diff
@@ -367,6 +378,48 @@ describe 'Robot.react DEPRECATED', ->
     @slackbot.robot.react matcher, @handleReaction
     listener = @slackbot.robot.listeners.shift()
     listener.matcher(@reactionMessage).should.be.false
+    
+describe 'Robot.fileShared', ->
+  before -> 
+    user = { id: @stubs.user.id, room: @stubs.channel.id }
+    @fileSharedMessage = { type: "file_shared", file_id: "F2147483862", file: { "id": "F2147483862"} }
```

Comment:
A small thing, but could you make this a `FileSharedMessage` instead of just representing it with JSON? Similar to how it's done for `react` and `presenceChange` tests (https://github.com/slackapi/hubot-slack/blob/ed09f71cadadc8906d7a683a93c78b4824053e3f/test/bot.coffee#L341)

Reply:
Hi @shanedewael thanks for the feedback, I've addressed this!